docs/slf4j1-bridge.md

```diff
@@ -41,6 +41,10 @@ import zio.logging.consoleJsonLogger
 val logger = Runtime.removeDefaultLoggers >>> consoleJsonLogger() >+> Slf4jBridge.initialize
 ```
 
+To enable log annotations and span propagation to underlying code which using SLF4J api, 
```

Comment:
Can we build the current fiber integration integration into an existing layer such as `Slf4jBridge.initialize`? It is great to have the ability to customize when necessary but it is also good to make common use cases easy and `enableCurrentFiber` is really an implementation detail where if the user is going to do something that requires it (basically propagating context from `FiberRef` values to `ThreadLocal` values we should just do it.

Reply:
i think we can put it to `Slf4jBridge.initialize` 
however there should be option to disable that as `enableCurrentFiber` may have impact to performance

which probably ends in something like

```scala
object Slf4jBridge {

  def initialize(propagateFibers: Boolean = true): ZLayer[Any, Nothing, Unit] = ???
```

Reply:
Makes sense. I might split it into two layers so it can be a `val` and you don't need the empty parentheses when you call it.

`
object Slf4jBridge {

  val initialize: ZLayer[Any, Nothing, Unit] =
    ???

  val initializeWithoutFiberRefPropagation: ZLayer[Any, Nothing, Unit] =
    ???
}
```

Reply:
@adamgfraser 

i added layers as you proposed, probably it is good that this propagation is enabled by default,
however it is also changing default behavior

Reply:
@justcoon Sounds good.My thought is that `EnableCurrentFiber` is necessary for integration between `FiberRef` and `ThreadLocal` values and that is exactly what you are asking us to do with the `Slf4jBridge` so that makes sense as the default and will give the "correct", though potentially not as performant value, so that should be the default. But if you think we should do it the other way definitely open to your thoughts on that.

Otherwise looks good to me subject to my comment about avoiding unnecessary ZIO workflows which we will handle separately.

Reply:
thank you very much for hints and review